dynamiqs/core/_utils.py

```diff
@@ -45,10 +45,7 @@ def get_solver_class(
 
 
 def compute_vmap(
-    f: callable,
-    cartesian_batching: bool,
-    is_batched: PyTree[bool],
-    out_axes: PyTree[int | None],
+    f: callable, cartesian_batching: bool, n_batch: [int], out_axes: PyTree[int | None]
 ) -> callable:
     # This function vectorizes `f` by applying jax.vmap over batched dimensions. The
```

Comment:
This inline comments should be updated throughout the function.

Reply:
I'm not sure what you mean but I updated the comment.